src/dnaio/_core.pyx

```diff
@@ -146,6 +150,27 @@ cdef class Sequence:
         return self.fastq_bytes(two_headers=True)
 
 
+cdef class SequenceBytes(Sequence):
```

Comment:
The naming is typically from most specific to least specific as in `MemoryError`, `FastqReader`, `EncodingWarning`, `PosixPath` etc. so I would call this a `BytesSequence`.

Reply:
Yes, that makes sense, as it is a specific type of sequence. Thanks. I will do that.